Tests/Marketplace/upload_packs.py

```diff
@@ -1148,6 +1148,11 @@ def main():
             pack.cleanup()
             continue
 
+        # remove XSIAM specific flags when not on XSIAM
+        if marketplace != XSIAM_MP:
+            if pack.is_data_source:
+                pack.is_data_source = False
```

Comment:
I would suggest doing it in the function `add_pack_type_tags` where we call the `is_data_source_pack` function, call it only for XSIAM.
```
if marketplace == XSIAM_MP:
    self._is_data_source = self.is_data_source_pack()
```